ci_framework/roles/libvirt_manager/tasks/deploy_layout.yml

```diff
@@ -50,6 +50,20 @@
   loop_control:
     label: "{{ item.key }}"
 
+- name: Find all public key in the default user directory
+  ansible.builtin.find:
+    paths: "{{ ansible_user_dir }}/.ssh"
+    file_type: file
+    patterns: '*.pub'
+  register: users_pub_key
+
+- name: Update authorized keys for all users
+  ansible.posix.authorized_key:
+    user: zuul
```

Comment:
that user is supposed to be on the hypervisor? It exists only in CI - others may have yet another username on the hypervisor.

Also, for dev case, they will probably get an ever growing authorized_keys at this point.